src/aleph/http/client.clj

```diff
@@ -725,7 +776,10 @@
        :exception-caught
        ([_ ctx ex]
         (when-not (d/error! d ex)
-          (log/warn ex "error in websocket client"))
+          (try
+            (error-logger ex)
+            (catch Throwable e
+              (log/warn "error in error logger" e))))
```

Comment:
Do you ignore stacktrace for the exception on purpose? What I mean is that `(log/warn e "error in error logger")` would print both the exception message and its trace, but your code will only print the message itself.